src/checkbox/index.tsx

```diff
@@ -58,9 +60,14 @@ export default class Checkbox extends PureComponent <CheckboxProps, CheckboxStat
   }))
 
   render() {
-    const { className, name, children, value, subLabel } = this.props
+    const { className, name, children, value, subLabel, labelBeforeInput } = this.props
+    const hasLabel = children || subLabel
```

Comment:
Is there a design where we can have a checkbox without label. Also is it really possible to have a sublabel without label ?

Reply:
I don't know, and hope that we never have a checkbox without a label, it's only a little change to make sure we don't serve empty markup in this event.
Having a subLabel only is possible with the current implementation so I didn't challenge it.